_overviews/jdk-compatibility/overview.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: multipage-overview
+title: Overview
+
+discourse: true
+
+partof: jdk-compatibility
+overview-name: JDK Compatibility
+
+num: 1
+permalink: /overviews/jdk-compatibility/:title.html
+---
+
+Scala runs primarily on the Java Virtual Machine (JVM). As Scala and the JVM improve independently over time, Scala drops compatibility with older versions of the Java Developer Kit (JDK) in order to focus development efforts on supporting new JVM features that benefit Scala.
+
+This table shows the first Scala release in each series that functions on each JDK.
+
+| JDK version | First Scala release supporting JDK version per series |
+|:-----------:|:-----------------------------------------------------|
+| 9           | 2.12.4, 2.11.12                                      |
+| 8           | 2.12.0, 2.11.0, 2.10.7                               |
+| 7           | 2.11.0, 2.10.0                                       |
+| 6           | 2.10.0                                               |
+
```

Comment:
I think this whole table may need a rethink.

There are two kinds of information to be conveyed: when support for a particular JDK was _added_, and when it was _dropped_.

Re support being _added_, to me the table seems to give the impression that 2.11.x doesn't work on Java 6. But it does.

Re support being _dropped_, that information is missing entirely. One of the most important jobs for this page to do, IMO, is convey that e.g. if you want to use Scala 2.12, you won't be able to use JDK 7 or less.

I'm don't actually know what the ideal format for conveying this information is, I'm just suggesting that it ought to be conveyed somehow. Whether in a single combined table somehow, or two tables, or what, idk.